package.json

```diff
@@ -104,6 +104,7 @@
     "start": "start-storybook -p 6006",
     "build-storybook": "build-storybook -o docs-build",
     "test-storybook": "test-storybook",
+    "prelint": "yarn install-czifui-locally",
```

Comment:
seems like `lint` needs `czifui` installed as well, so I'm adding `prelint`. Does that sound right? Not sure why this wasn't needed before this PR 🤔

Reply:
Aah, thanks for adding this. 👍🏻